src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -304,8 +279,5 @@ export default compose(
         onfidoToken: {
             key: ONYXKEYS.ONFIDO_TOKEN,
         },
-        reimbursementAccountDraft: {
```

Comment:
I think you need it? (used by ReimbursementAccountUtils.getDefaultStateForField)